docs_user/assemblies/assembly_migrating-ceph-rbd.adoc

```diff
@@ -13,31 +15,22 @@ the Compute nodes for an HCI environment or dedicated storage nodes.
 To migrate Red Hat Ceph Storage Rados Block Device (RBD), your environment must
 meet the following requirements:
 
-* {Ceph} is running version 6 or later and is managed by cephadm.
-* NFS Ganesha is migrated from a {OpenStackPreviousInstaller}-based
-  deployment to cephadm. For more information, see
-  xref:creating-a-ceph-nfs-cluster_migrating-databases[Creating a NFS Ganesha
+* {Ceph} is running version 6 or later and is managed by `cephadm`.
+* NFS Ganesha is migrated from a {OpenStackPreviousInstaller} deployment to `cephadm`. For more information, see xref:creating-a-ceph-nfs-cluster_migrating-databases[Creating a NFS Ganesha
   cluster].
-* Both the {Ceph} public and cluster networks are propagated, with
-  {OpenStackPreviousInstaller}, to the target nodes.
-* Ceph MDS, Ceph Monitoring stack, Ceph MDS, Ceph RGW and other services are
-  migrated to the target nodes.
+* Both the {Ceph} public and cluster networks are propagated with
+  {OpenStackPreviousInstaller} to the target nodes.
+* Ceph Metadata Server, monitoring stack, Ceph Ceph Object Gateway, and other services are migrated to the target nodes.
```

Comment:
Does “other services” refer to other Red Hat Ceph Storage services, specifically?

Reply:
"Other services" is a little vague (at least to me). But your explanation would work, e.g., "Ceph Metadata Server, monitoring stack, Ceph Object Gateway, and any other daemon that is deployed on Controller nodes."

Good catch on the double "Ceph" too.